Scribe/SettingsTab/SettingsViewController.swift

```diff
@@ -102,7 +100,7 @@ final class SettingsViewController: UIViewController {
       parentTable.tableFooterView?.isHidden = false
     }
 
-    footerButton.setTitle("Install keyboards", for: .normal)
+    footerButton.setTitle("Install keyboards", for: .normal) // is this ever used?
```

Comment:
This button appears on the settings screen if the user doesn't have an installed keyboard, @Jag-Marcel :) I'll remove the comment 😊